source/common/assets/less/geometry/codemirror.less

```diff
@@ -32,7 +32,7 @@
 
   &.CodeMirror-empty {
     // If the editor is empty, display a nice background image
-    background-image: url('../img/feather.svg');
+    background-image: url('../../img/feather.svg');
```

Comment:
As moving files around result in a large change, I'm wondering if it is possible to separate out moving the Less files under the `source` directory from the webpack (Electron Forge) change, to make this PR smaller and easier to review. What do you think?

Reply:
Sadly not, at least not easily without replicating what webpack does. The less files previously were compiled to css by hand, and then put into a `source/renderer/assets/css`. All references to other files are relative to this folder. Webpack requires that the references are relative to the original file.

Reply:
Phew, at least for once it's good that I don't make use of all too many external resources …